i18n/plk/src/preset.i18n.json

```diff
@@ -9,8 +9,8 @@
 	"default.test.preset": "[Domyślna]",
 	"default.test.preset.description": "Pusty predefiniowany test, które nie dodaje żadnych argumentów",
 	"no.cl.arch": "Predefiniowana konfiguracja {0}: nie określono architektury dla elementu cl.exe, używając domyślnej opcji x86",
-	"no.cl.toolset.arch": "Skonfiguruj ustawienie wstępne {0}: nie określono zestawu narzędzi dla elementu cl.exe, używając domyślnej opcji x86",
-	"invalid.cl.toolset.arch": "Skonfiguruj ustawienie wstępne {0}: Określono nieoczekiwaną architekturę zestawu narzędzi \"{1}\" czy chodziło Ci o \"host ={1}\"?",
+	"no.cl.toolset.arch": "Skonfiguruj ustawienie wstępne {0}: nie określono zestawu narzędzi dla elementu cl.exe, używając domyślnej opcji host=x86",
```

Comment:
weird, this is not updated

Reply:
I added "host=" to that line, but it's not highlighted for some reason.